core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -160,6 +162,21 @@ class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
     set("spark.serializer", classOf[KryoSerializer].getName)
     this
   }
+  /**
+   * Use Kryo serialization and register the given set of Avro schemas so that the generic
+   * record serializer can decrease network IO
+   */
+  def registerAvroSchema(schemas: Array[Schema]): SparkConf =
```

Comment:
how about using varargs here? `def registerAvroSchema(schemas: Schema*): SparkConf`

then you could call using any of:
```
registerAvroSchema(schema)
registerAvroSchema(schemaOne, schemaTwo, schemaThree)
registerAvroSchema(schemaSeq: _*) // works w/ an array too
```

and maybe rename the method to plural, `registerAvroSchemas`